index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Peter's Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="statusBar">
+              <img src="./assets/StatusBar.png" alt="">
+            </div>
+            <div class="nav-bar">
+              <div class="logo-placeholder"></div>
+              <div class="menu-button">
+                <img src="./assets/Icon - Menu.png" alt="">
+              </div>
+            </div>
+        </nav>
+    </header>
+    <main>
```

Comment:
- [optional] it would have still been ok to use the section tag here on its own without the main tag. Section tag defines the section of documents and divides the content into section and subsections. also an advantage of the section tag is, it is a semantic element, which describes its meaning to both browser and developer while  The content inside the <main> element should be unique to the document.

Reply:
So will it be correct practice if I add main or remove the section element until when necessary?

Reply:
@Peter-Kibet  you can use just the section tag for that headline section as It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.